From: =?UTF-8?q?Carl=20F=C3=BCrstenberg?= <carl@excito.com>
Date: Wed, 4 May 2011 15:06:36 +0200
Subject: Disabling the error header code

Disabling the activation of error header code as it prevents further
moditications of the header/body in certain situations
---
 system/core/Exceptions.php |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/system/core/Exceptions.php b/system/core/Exceptions.php
index f565956..1dc11c4 100755
--- a/system/core/Exceptions.php
+++ b/system/core/Exceptions.php
@@ -119,7 +119,7 @@ class CI_Exceptions {
 	 */
 	function show_error($heading, $message, $template = 'error_general', $status_code = 500)
 	{
-		set_status_header($status_code);
+		//set_status_header($status_code);
 
 		$message = '<p>'.implode('</p><p>', ( ! is_array($message)) ? array($message) : $message).'</p>';
 
@@ -175,4 +175,4 @@ class CI_Exceptions {
 // END Exceptions Class
 
 /* End of file Exceptions.php */
-/* Location: ./system/core/Exceptions.php */
\ No newline at end of file
+/* Location: ./system/core/Exceptions.php */
-- 
